components/binding-usage-controller/internal/controller/usagekind/protection_controller.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 const (
-	finalizerName = "servicecatalog.kyma.cx/usage-kind-protection"
+	finalizerName = "servicecatalog.kyma-project.io/usage-kind-protection"
```

Comment:
**suggestion**

What do you think about this approach?

```
var finalizerName = fmt.Sprintf("%s/usage-kind-protection", v1alpha1.SchemeGroupVersion.Group)
```

thanks to that in future we will not maintain that, what do u think?

Reply:
I believe that we won't change the domain name again. But you never know how kyma will look like tomorrow, so I will apply your suggestion 👍